src/index.ts

```diff
@@ -76,4 +77,70 @@ app.post("/logout", async(c) => {
   return new Response();
 });
 
+app.post("/attendance", async(c) => {
+  /*ネットワークの確認を行う*/
+  const allowPattern = new RegExp(c.env.ALLOW_IP_REGEX);
+  if ( !allowPattern.test(c.req.header("cf-connectiong-ip") as string) ) {
+    return c.text("Forbidden", 403);
+  }
+
+  /*学生の認証を行う*/
+  //sessionがcookieあるか確認
+  const session = c.get("session");
+  if ( !session ){
+    return c.text("Unauthorized", 401);
+  }
+  //session情報がsessionテーブルにある確認
+  const existResult = await c.env.DB
+  .prepare("SELECT * FROM session WHERE id = ?")
+  .bind(session['id'])
```

Comment:
```suggestion
  .bind(session.id)
```
これはこう書いても全く同じ意味になります. 他の箇所も同様にご対応お願いします.